README.md

```diff
@@ -83,7 +85,10 @@ If you are a complete Docker newbie, you should probably follow the [series of t
 
 ## Containers
 
-[Your basic isolated Docker process](http://etherealmind.com/basics-docker-containers-hypervisors-coreos/).  Containers are to Virtual Machines as threads are to processes.  Or you can think of them as chroots on steroids.
+- [Your basic isolated Docker process](http://etherealmind.com/basics-docker-containers-hypervisors-coreos/).
+- Containers are writable and ephemeral.
+- Containers are created from images.
```

Comment:
"Containers are created from images" means nothing by itself to someone who doesn't already know what images are.